src/css/components/typography-styles.css

```diff
@@ -0,0 +1,162 @@
+/**
+  Decanter typography styles
+*/
+
+@layer components {
+  .types
```

Comment:
```suggestion
  .types,
```

Reply:
Oh I haven't cleaned up a lot of the stuff - some of it I have AI moved things around 😂 I'll go back and see if I can rewrite the modular type stuff in a cleaner way too.
What are you doing here by the way @sherakama ? 👀